include/bm/bm_sim/named_p4object.h

```diff
@@ -24,6 +24,8 @@
 #define BM_BM_SIM_NAMED_P4OBJECT_H_
 
 #include <string>
+#include <sstream>
```

Comment:
It used to be there because of a use of stringstream in that file.  I moved it to another file where stringstream is currently being used.